onnxscript/analysis.py

```diff
@@ -145,24 +143,20 @@ def visitBlock(block, live_out):
         if isinstance(stmt, ast.FunctionDef):
             return live_out
         try:
-            if stmt.value.func.id == "print":
+            if stmt.value.func.id == "print":  # type: ignore[attr-defined]
                 # Any call to print function are ignored.
                 return live_out
         except (TypeError, AttributeError):
```

Comment:
nit: from CodeQL: 'except' clause does nothing but pass and there is no explanatory comment.

Helpful to add a comment?

Reply:
Cleaned up, thanks!